schema/DoctrineMigrations/Version20231218161807.php

```diff
@@ -0,0 +1,108 @@
+                                            \'text/plain\'
+                                        )'
+        );
+
+        $this->addSql('INSERT INTO `NotificationTemplatesContents` (
+                                        `fromName`,
+                                        `fromAddress`,
+                                        `subject`,
+                                        `body`,
+                                        `notificationTemplateId`,
+                                        `languageId`,
+                                        `bodyType`
+                                        ) VALUES (
+                                            \'Notificaciones IvozProvider\',
+                                            \'no-reply@ivozprovider.com\',
+                                            \'Credenciales de acceso IvozProvider\', 
+                                            \'A continuación se le indican las credenciales de acceso para el portal de administración de Ivoz Provider:\n Usuario: ${USER}\n Contraseña: ${PASSWORD} \n\', 
+                                            (SELECT id FROM NotificationTemplates WHERE type=\'accessCredentials\'),
+                                            3, 
+                                            \'text/plain\'
+                                        )'
+        );
+
+        $this->addSql('INSERT INTO `NotificationTemplatesContents` (
+                                        `fromName`,
+                                        `fromAddress`,
+                                        `subject`,
+                                        `body`,
+                                        `notificationTemplateId`,
+                                        `languageId`,
+                                        `bodyType`
+                                        ) VALUES (
+                                            \'IvozProvider Notifications\',
+                                            \'no-reply@ivozprovider.com\',
+                                            \'IvozProvider Access Credentials\', 
+                                            \'Below are the access credentials for the administration portal of Ivoz Provider:\n User: ${USER}\n Password: ${PASSWORD} \n\', 
+                                            (SELECT id FROM NotificationTemplates WHERE type=\'accessCredentials\'),
+                                            4, 
+                                            \'text/plain\'
+                                        )'
+        );
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE NotificationTemplates CHANGE type type VARCHAR(25) NOT NULL COMMENT \'[enum:voicemail|fax|limit|lowbalance|invoice|callCsv|maxDailyUsage]\'');
+        $this->addSql('DELETE FROM NotificiationTemplates WHERE  type=\'accessCredentials\'');
```

Comment:
This delete is not possible after the alter table, it must be done before, otherwise the field used in the where clause can not have that value.